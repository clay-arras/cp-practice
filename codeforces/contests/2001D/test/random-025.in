1
22
21 9 7 5 12 16 12 9 9 5 18 16 13 20 3 4 16 23 17 3 20 23
