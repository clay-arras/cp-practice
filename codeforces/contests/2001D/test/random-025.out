13
21 7 16 12 9 5 18 13 20 3 4 17 23
